README.md

```diff
@@ -10,6 +8,35 @@ a GitHub Integration built with [probot](https://github.com/probot/probot) that
 
 See [docs/deploy.md](docs/deploy.md) if you would like to run your own instance of this plugin.
 
+## Modes of operations
+
+### Default
+
+By default, Probot DCO enforces the presence of [valid DCO signoffs](#how-it-works) on all commits (excluding bots and merges). If a PRs contains commits that lack a valid Signed-off-by line, they are blocked until a correctly signed-off revision of the commit is pushed. This closely mirrors the upstream Linux kernel process.
+
+### Individual remediation commit support
```

Comment:
Can we add instructions on how to add remediation commits using `git` and the GitHub Web UI? I know you add instructions to the check runs which is great, but I think we should document it here as well, what do you think?

If I understand it correctly, ideally remediation commits would be empty commits with the correct messages, but there is no way to add empty commits using the GitHub Web UI as far as I know.